### ISSUE TITLE
8.3.0 docker image fails to start

### ISSUE THREAD

**Issue by Dup4** [DOMjudge/domjudge#2598]
### Description of the problem
> Replace this line with a short description.

```bash
docker run \
    -d \
    --restart=always \
    --name="${NAME}" \
    -e CONTAINER_TIMEZONE="${TZ}" \
    -e WEBAPP_BASEURL=/domjudge \
    -e MYSQL_HOST="${MYSQL_HOST}" \
    -e MYSQL_PORT="${MYSQL_PORT}" \
    -e MYSQL_DATABASE="${MYSQL_DATABASE}" \
    -e MYSQL_USER="${MYSQL_USER}" \
    -e MYSQL_PASSWORD="${MYSQL_PASSWORD}" \
    -e MYSQL_ROOT_PASSWORD="${MYSQL_PASSWORD}" \
    -p "${DOMSERVER_PORT}":80 \
    "domjudge/domserver:8.3.0"
```

Use this command to start the image

![ccd40ecf5e798f406ac7c848eaaba33a](https://github.com/DOMjudge/domjudge/assets/38343778/9ead5cd2-32de-43ac-82c6-6ffa5302eb74)

It's failing to start.

![bc86760ef8188efed19a99faeb9ffeb9](https://github.com/DOMjudge/domjudge/assets/38343778/290b4e77-6d21-47f7-adc2-7742d8990499)

And I checked, and there are no target file in the corresponding directory.

https://github.com/DOMjudge/domjudge-packaging/blob/main/docker/domserver/scripts/start.d/50domjudge.sh#L227

I'm guessing the crash was here at startup.

And I'm guessing that the reason for this is that the `symfony/runtime` was not included in the build image.

### Your environment
> Include details about your installation here.
> - DOMjudge version (e.g. 7.0.0 or a github commit hash)
> - Operating system / Linux distribution and version (e.g. Ubuntu 18.04)
> - Webserver (e.g. Apache or nginx)

- DOMjudge: 8.3.0
- OS: Ubuntu 22.04

**Comment by vmcj** [comment]
We are aware of the issue indeed, I already pushed to the latest tag back for the earlier image (which is why you encounter this only when you explicit pick that container).

**Comment by Dup4** [comment]
Okay, I'll workaround with the latest version first.

**Comment by Dup4** [comment]
<img width="776" alt="image" src="https://github.com/DOMjudge/domjudge/assets/38343778/0addebb4-fc4a-43a3-8095-89fb98a41aed">

This could be one of the clues

**Comment by Dup4** [comment]
https://github.com/Dup4/domjudge-packaging/blob/main/docker/domserver/Dockerfile#L5

After adding this environment variable and rebuilding the image, I was able to get 8.3.0 up and running!

**Comment by vmcj** [comment]
> https://github.com/Dup4/domjudge-packaging/blob/main/docker/domserver/Dockerfile#L5
> 
> After adding this environment variable and rebuilding the image, I was able to get 8.3.0 up and running!

Yes, but that's not the fix I want in the end (do still PR it though). I prefer we would have detected this in our CI before and would have fixed that composer never runs as root.
